### ISSUE TITLE
Publish the current version to Brackets Registry

### ISSUE THREAD

**Issue by petetnt** [DennisKehrig/brackets-show-whitespace#51]
I noticed that the current version according to `package.json` is `"version": "2.1.0"`, while the version on the registry is `2.0.1`:

```
Show Whitespace 2.0.1
by Dennis Kehrig / DennisKehrig on 2014-05-29 
Visualizes whitespace similar to how Sublime Text does it
```

Unless there's something blocking this, I see that @DennisKehrig hasn't been online for a bit so that might complicate things a bit (unless changing the package name a bit is okay)

**Comment by le717** [comment]
I've been given basically total control over the extension,: and believe me, I've long wanted to push a new version. Problem is the Brackets registry only allows one person to manage an extension. That person would be Dennis, and as you correctly noted, he's been inactive for a while (I'd have to find the last comment he left for the exact length of time).

As for changing the name a slight bit, I have considered that too, but in the eyes of the registry that would be considered a new extension, not an update. This extension is (last I checked) one of the top 20 most popular extensions. I'd be much more visible if this could be updated instead of making a new one. Further, if I pushed a new copy of this extension, then why am I still developing on here? Why not  fork it and continue work on my own GitHub account?

So yea. That's the deal.

**Comment by petetnt** [comment]
Yup, figured out as much... One way could be if the ownership could be moved to you manually, considering that you have a written clause of having total control. But it's understandably kinda hard pickle to solve (unless the clause explicitly states the state of registry too). 

Losing the existing users would be a shame, I agree on that too.

**Comment by le717** [comment]
Found it. #26, last post on January 15, 2015.

By this point it might be more advantageous to introduce a new fork...

**Comment by le717** [comment]
Alright, I looked up the comment where I was given control. https://github.com/DennisKehrig/brackets-show-whitespace/issues/26#issuecomment-68696516

> Hey @le717, I'm happy to hear it! I added you as a contributor. I won't have any time to work on this for the upcoming weeks (moving halfway across the globe), so for the time being, you are in complete control. :)

I noted in the very next comment (https://github.com/DennisKehrig/brackets-show-whitespace/issues/26#issuecomment-68733718) lack of registry access, but it was too late. Additionally, Dennis implied he'd only be gone for a few weeks, not over a year.

I don't really know how this could be interpreted. "Complete control" could mean "including registry", but "upcoming weeks" and "for the time being" imply "until I get back and can push updates".

It'd help if the registry supported multiple publishers... :(

**Comment by pmario** [comment]
@le717 Hi, Is this project still alive? 

I did play around a little bit with css settings. Found out, that it would be possible to create vertical bars similar to "indent guides" using tabs. ... Spaces may be possible too.

Wanted to create a PR, but saw this issue. ... I think the AddOn should be forked. 

License is MIT so no problem there.

**Comment by le717** [comment]
Hey @pmario!

I've obviously not made any changes to the repo, but if you'd like to submit a PR, I'd be more than happy to look at it and merge it in. As far as forking, everything stated in this thread still stands, including forking. Brackets doesn't have this functionality out of the box (it'd be good if they did), but this is still the #1 whitespace extension. I'd much rather get an update pushed. I'm going to try to reach out to the Brackets repo to see if I can get ownership rights (perhaps something has changed) and if not, if the extension can be adopted into the core application.
